core/templates/dev/head/domain/exploration/EditableExplorationBackendApiServiceSpec.js

```diff
@@ -180,14 +180,16 @@ describe('Editable exploration backend API service', function() {
       exploration.title = 'New Title';
       exploration.version = '2';
 
-      $httpBackend.expect('PUT', '/createhandler/data/0').respond(
-        exploration);
+      $httpBackend.expect('PUT', '/createhandler/data/0')
+        .respond(exploration);
 
       // Send a request to update exploration
-      EditableExplorationBackendApiService.updateExploration(
-        exploration.exploration_id, exploration.version,
-        exploration.title, []).then(
-        successHandler, failHandler);
+      EditableExplorationBackendApiService
+        .updateExploration(
```

Comment:
Seems odd to break at dot here, previous version was better

Reply:
Done.